packages/proxy/src/providers/openai.ts

```diff
@@ -66,3 +73,60 @@ export function makeFakeOpenAIStreamTransformer() {
     },
   });
 }
+
+export async function normalizeOpenAIMessages(
+  messages: ChatCompletionMessageParam[],
+): Promise<ChatCompletionMessageParam[]> {
+  return Promise.all(
+    messages.map(async (message) => {
+      console.log("message", message);
+      if (
+        message.role === "user" &&
+        message.content &&
+        typeof message.content !== "string"
+      ) {
+        message.content = await Promise.all(
+          message.content.map(
+            async (c): Promise<ChatCompletionContentPart> =>
+              await normalizeOpenAIContent(c),
+          ),
+        );
+      }
+      return message;
+    }),
+  );
+}
+
+// https://platform.openai.com/docs/guides/pdf-files?api-mode=chat
+async function normalizeOpenAIContent(
+  content: ChatCompletionContentPart,
+): Promise<ChatCompletionContentPart> {
+  if (typeof content === "string") {
+    return content;
+  }
+  switch (content.type) {
+    case "image_url":
+      console.log("image_url", content.image_url.url);
```

Comment:
Likely need to remove this, https://github.com/braintrustdata/braintrust-proxy/pull/209/files#diff-680db4418ead323337b7b031e9eb51c25ec7d09fae027b6a247f2685c819ff5fR82, https://github.com/braintrustdata/braintrust-proxy/pull/209/files#diff-680db4418ead323337b7b031e9eb51c25ec7d09fae027b6a247f2685c819ff5fR118

Reply:
whoops thank you